server/utils/placementRequests/adminIdentityBar.ts

```diff
@@ -26,33 +27,38 @@ export const adminActions = (
   user: UserDetails,
 ): Array<IdentityBarMenuItem> => {
   if (placementRequest.status === 'matched' && placementRequest.booking) {
-    const bookingType = placementRequest.booking.type
-    const matchedActions =
-      bookingType === 'space'
-        ? [
-            {
-              href: managePaths.premises.placements.changes.new({
-                premisesId: placementRequest.booking.premisesId,
-                placementId: placementRequest.booking.id,
-              }),
-              text: 'Change placement',
-            },
-          ]
-        : [
-            {
-              href: managePaths.bookings.dateChanges.new({
-                premisesId: placementRequest.booking.premisesId,
-                bookingId: placementRequest.booking.id,
-              }),
-              text: 'Amend placement',
-            },
-          ]
+    const matchedActions = []
+
+    if (placementRequest.booking.type === 'legacy') {
+      matchedActions.push({
+        href: managePaths.bookings.dateChanges.new({
+          premisesId: placementRequest.booking.premisesId,
+          bookingId: placementRequest.booking.id,
+        }),
+        text: 'Amend placement',
+      })
+    }
+
+    if (
+      placementRequest.booking.type === 'space' &&
+      ['upcoming', 'arrived'].includes(overallStatus(placementRequest.spaceBookings[0]))
+    ) {
+      matchedActions.push({
+        href: managePaths.premises.placements.changes.new({
+          premisesId: placementRequest.booking.premisesId,
+          placementId: placementRequest.booking.id,
+        }),
+        text: 'Change placement',
```

Comment:
I'm wondering why the difference in wording between the two types - 'Change' vs 'Amend' seems to indicate nothing! It would be nice if we could standardise on terminology e.g. 'booking' for a legacy booking and 'placement' for a `space-booking` but I know the terminology throughout the service is confused.